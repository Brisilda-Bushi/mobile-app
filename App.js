import * as React from "react";
import { Provider } from 'react-redux';
import { Store } from "./redux/store";
import HomeScreen from "./screens/HomeScreen";

export default function App() {
  return (
    <Provider store={Store}>
    <HomeScreen />
   </Provider>
  );
}

