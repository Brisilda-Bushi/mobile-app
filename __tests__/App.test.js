import React from 'react';
import renderer from 'react-test-renderer';
import App from '../App';

jest.mock('react-native/Libraries/Animated/NativeAnimatedHelper');

test('renders correctly', () => {
  const tree = renderer.create(<App />).toJSON();
  expect(tree).toMatchSnapshot();
});