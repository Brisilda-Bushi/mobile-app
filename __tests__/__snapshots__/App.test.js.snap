// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<RCTSafeAreaView
  emulateUnlessSupported={true}
  style={
    Object {
      "flex": 1,
      "top": "5%",
    }
  }
>
  <Image
    source={
      Object {
        "uri": "https://images.unsplash.com/photo-1519750783826-e2420f4d687f?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxleHBsb3JlLWZlZWR8Mnx8fGVufDB8fHx8&w=1000&q=8",
      }
    }
    style={
      Object {
        "bottom": 0,
        "left": 0,
        "position": "absolute",
        "right": 0,
        "top": 0,
      }
    }
  />
  <View>
    <Text
      style={
        Object {
          "color": "white",
          "display": "flex",
          "flexDirection": "row",
          "fontSize": 22,
          "fontWeight": "bold",
          "padding": 10,
          "textAlign": "center",
        }
      }
      testID="myText"
    >
      Mobile App Listing
    </Text>
  </View>
  <RCTScrollView
    contentContainerStyle={
      Object {
        "padding": 20,
      }
    }
    data={Array []}
    disableVirtualization={false}
    getItem={[Function]}
    getItemCount={[Function]}
    horizontal={false}
    initialNumToRender={10}
    keyExtractor={[Function]}
    maxToRenderPerBatch={10}
    onContentSizeChange={[Function]}
    onEndReachedThreshold={2}
    onLayout={[Function]}
    onMomentumScrollEnd={[Function]}
    onScroll={[Function]}
    onScrollBeginDrag={[Function]}
    onScrollEndDrag={[Function]}
    removeClippedSubviews={false}
    renderItem={[Function]}
    scrollEventThrottle={0.0001}
    stickyHeaderIndices={Array []}
    style={Object {}}
    updateCellsBatchingPeriod={50}
    viewabilityConfigCallbackPairs={Array []}
    windowSize={21}
  >
    <View />
  </RCTScrollView>
</RCTSafeAreaView>
`;
